serverless_llm/store/tests/python/test_save_model.py

```diff
@@ -0,0 +1,73 @@
+# ---------------------------------------------------------------------------- #
+#  serverlessllm                                                               #
+#  copyright (c) serverlessllm team 2024                                       #
+#                                                                              #
+#  licensed under the apache license, version 2.0 (the "license");             #
+#  you may not use this file except in compliance with the license.            #
+#                                                                              #
+#  you may obtain a copy of the license at                                     #
+#                                                                              #
+#                  http://www.apache.org/licenses/license-2.0                  #
+#                                                                              #
+#  unless required by applicable law or agreed to in writing, software         #
+#  distributed under the license is distributed on an "as is" basis,           #
+#  without warranties or conditions of any kind, either express or implied.    #
+#  see the license for the specific language governing permissions and         #
+#  limitations under the license.                                              #
+# ---------------------------------------------------------------------------- #
+import unittest
+import os
+import shutil
+import torch
+from transformers import AutoModelForCausalLM
+from serverless_llm_store import save_model
+
+
+class TestSaveModelIntegration(unittest.TestCase):
+    def setUp(self):
+        # Set up a temporary directory for the test
+        self.model_name = "facebook/opt-1.3b"
```

Comment:
`facebook/opt-1.3b` may be too large, I suppose just use `facebook/opt-125m` may be better.

Reply:
OK, sounds good, will modify here.